frontend/src/components/Movies/Movies.component.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+  <v-layout row>
+    <v-flex
+      xs12
+      sm5
+      md3
+      offset-lg2
+    >
+      <v-form
+        v-model="valid"
+      >
+        <v-text-field
+          v-model="title"
+          :rules="titleRules"
+          :counter="255"
+          label="Title"
+          required
+        />
+        <v-text-field
+          :counter="255"
+          :rules="genreRules"
+          v-model="genre"
+          label="Genre"
+          required
+        />
+        <v-text-field
+          :counter="255"
+          v-model="director"
+          label="Director"
+        />
+        <v-text-field
+          :counter="255"
+          v-model="actors"
+          label="Actors"
+        />
+        <v-text-field
+          :counter="255"
+          v-model="duration"
+          label="Duration"
+        />
+        <v-text-field
+          :counter="255"
+          v-model="description"
+          label="Description"
+        />
+        <v-btn
+          :disabled="!valid"
+          @click="submit"
+        >
+          add
+        </v-btn>
+        <v-btn @click="clear">clear</v-btn>
+      </v-form>
+    </v-flex>
+    <v-flex
+      s12
+      sm5
+      md5
+      offset-lg2
+    >
+      <v-list
+        two-line
+        subheader
+      >
+        <v-subheader>Movies</v-subheader>
+        <v-list-tile>
+          <v-text-field
+            v-model="search"
+            append-icon="search"
+            label="Search"
+            class="mx-3"
+          />
+        </v-list-tile>
+        <v-list-tile
+          v-for="movie in filteredMovies"
+          :key="movie.id" >
+          <v-list-tile-content>
+            <v-list-tile-title>{{ movie.title }}</v-list-tile-title>
+            <v-list-tile-sub-title>{{ movie.genre }}</v-list-tile-sub-title>
+          </v-list-tile-content>
+          <v-list-tile-action >
+            <v-btn
+              icon
+              ripple
+              @click="remove(movie.id)"
+            >
+              <v-icon color="grey lighten-1">delete</v-icon>
+            </v-btn>
+          </v-list-tile-action>
+        </v-list-tile>
+      </v-list>
+    </v-flex>
+  </v-layout>
+
+</template>
+
+<script>
+import MoviesController from 'Controllers/movies.controller';
+
+export default {
+  data: () => ({
```

Comment:
ovaj deo se ponavlja 3 puta i dugacak je... aj da zamenimo to sve sa modelom, na pocetku ga instaniciramo praznog